### ISSUE TITLE
Building not working (Grunt error)

### ISSUE THREAD

**Issue by Hug0Vincent** [kakawait/hugo-tranquilpeak-theme#315]
## Problem
Hi, tanks for sharing your theme, I would like to change the font so I modify the file in src/scss/utils/_variables.scss and src/scss/utils/_fonts.scss and after I tried to build but I have an errror :
 
````python
test@test: ~/Desktop/user/test/themes/hugo-tranquilpeak-theme grunt buildProd
Loading "Gruntfile.js" tasks...ERROR
>> TypeError [ERR_INVALID_CALLBACK]: Callback must be a function
Warning: Task "buildProd" not found. Use --force to continue.

Aborted due to warnings.
````

## Steps to reproduce the behavior
just run grunt buildProd or grunt default

**Comment by GioCC** [comment]
Same problem here, using both Node.js 10.5 and 11.8 (on Win7-x64 and Win10-x64), and with any grunt task. Anybody has a hint?

**Comment by GioCC** [comment]
Launching grunt with the -v option, an error can be traced to file "tranquilpeak\node_modules\rand-token\index.js".
It seems that at the switch(options.source), case 'default' (line 57) the assignment of 'options.source' causes  some trouble. Since I gathered that "crypto.pseudoRandomBytes()" might have some deprecation problems,  I tentatively replaced the instruction with the same one used for case 'crypto' (i.e. "options.source = crypto.randomBytes;").
Now the execution of "grunt default" proceeds without errors...  that is, until it gets stuck forever with no error messages at a point where it lists "Watching src\scss\utils\mixins\_tag.scss for changes."

**Comment by August1914** [comment]
running `grunt` on new site install, same error:
  `Loading "Gruntfile.js" tasks...ERROR`
  `>> SyntaxError: Unexpected token case`
  `Warning: Task "default" not found. Use --force to continue.`
npm 11.8
grunt-cli 1.3.2
grunt 0.4.5
osx 10.14

running `grunt buildProd` or `grunt default'  returns
`Warning: Task "buildProd" not found. Use --force to continue.`

Any clues on working through this appreciated

**Comment by sherry0429** [comment]
same eror again.

**Comment by raztiff** [comment]
Same errors here... Managed to get it working though after a quick bit of messing around.


```
Loading "Gruntfile.js" tasks...ERROR
>> TypeError [ERR_INVALID_CALLBACK]: Callback must be a function
>>     at Object.randomBytes (internal/crypto/random.js:48:11)
>>     at Object.options.source (../themes/hugo-tranquilpeak-theme/node_modules/rand-token/index.js:60:25)
>>     at Object.generate (../themes/hugo-tranquilpeak-theme/node_modules/rand-token/index.js:95:29)
>>     at Object.module.exports [as cssmin] (.../themes/hugo-tranquilpeak-theme/tasks/config/cssmin.js:12:30)
>>     at invokeConfigFn (../themes/hugo-tranquilpeak-theme/Gruntfile.js:41:24)
>>     at Object.module.exports (../themes/hugo-tranquilpeak-theme/Gruntfile.js:51:3)
>>     at loadTask (../themes/hugo-tranquilpeak-theme/node_modules/grunt/lib/grunt/task.js:325:10)
>>     at Task.task.init (../themes/hugo-tranquilpeak-theme/node_modules/grunt/lib/grunt/task.js:437:5)
>>     at Object.grunt.tasks (../themes/hugo-tranquilpeak-theme/node_modules/grunt/lib/grunt.js:120:8)
>>     at Liftoff.<anonymous> (../.nvm/versions/node/v10.13.0/lib/node_modules/grunt-cli/bin/grunt:66:13)

No tasks specified, running default tasks.
Running tasks: default
Warning: Task "default" not found. Use --force to continue.
```

I quickly just for testing purposes removed the references of the `rand-token` from `tasks/config/cssmin.js` and `tasks/config/uglify.js` which got further in the build process, but failed at `sync:dev` task as follows:

```
> hugo-tranquilpeak-theme@0.4.4-BETA start ../themes/hugo-tranquilpeak-theme
> grunt default

Running "clean:build" (clean) task
Cleaning static...OK

Running "sass:dev" (sass) task

Running "concat:devJs" (concat) task
File static/js/tranquilpeak.js created.

Running "sync:dev" (sync) task
Warning: Cannot read property 'bigint' of undefined Use --force to continue.

Aborted due to warnings.
``` 

Little search around this could be some incompatibilities around grunt and newer version of node, so I attempted to fall back on recommendations to `10.4.1`, this worked!

```
> hugo-tranquilpeak-theme@0.4.4-BETA start /Users/ryan/Sites/rst-tech-blog/themes/hugo-tranquilpeak-theme
> grunt default

Running "clean:build" (clean) task
Cleaning static...OK

Running "sass:dev" (sass) task

Running "concat:devJs" (concat) task
File static/js/tranquilpeak.js created.

Running "sync:dev" (sync) task

Running "sails-linker:devJs" (sails-linker) task
padding length 0
File "layouts/partials/script.html" updated.

Running "sails-linker:devCss" (sails-linker) task
padding length 4
File "layouts/partials/head.html" updated.

Running "replace:linker" (replace) task

Running "replace:cssTranquilpeak" (replace) task

Running "watch" task
Waiting...
```

**Comment by kakawait** [comment]
fixed via #351 (work on latest node on my side)
